README.md

```diff
@@ -40,3 +40,19 @@ Contributions are very welcome, as are feature requests and suggestions. Please
 [codecov-url]: https://codecov.io/gh/JuliaData/CSV.jl
 
 [issues-url]: https://github.com/JuliaData/CSV.jl/issues
+
+## Alternatives
+
+There are several other packages for reading CSV files in Julia, which may suit your needs better:
+
+* The standard library contains [DelimitedFiles.jl](https://docs.julialang.org/en/v1/stdlib/DelimitedFiles/),
+  which is perfect for quickly reading small files.
```

Comment:
Good point. Maybe something like this:
```suggestion
* The standard library contains [DelimitedFiles.jl](https://docs.julialang.org/en/v1/stdlib/DelimitedFiles/).
  This returns a `Matrix` rather than a `DataFrame`, thus works best for files of homogenous element type. 
  On large files, CSV.jl will be much faster.
```

Reply:
A couple more notes:

CSV doesn't return a DataFrame by default, it returns a CSV.File, so we should be careful not to imply it does

DelimitedFiles won't be a stdlib in upcoming Julia versions, I think

Reply:
Looks good.

Reply:
> DelimitedFiles won't be a stdlib in upcoming Julia versions 
That's slightly horrifying